### DISCUSSION TITLE
Stability Problems on Galaxy S5

### DISCUSSION THREAD

**Discussion by ulph0** [osmandapp/OsmAnd#20751] [Q&A] [answered]
hi,

since a year or so, osmand~ is nearly unsusable for me. i am running an galaxy s5 lineageOS and have my cards on a pretty ok sandisk ultra card. osmand crashes always. only sometimes obn very short routes it doesn't crash. since i need a reliable software i switched to organic maops but there thje routes an functions of course aren't that good. are there any hints for very old / slow mobiles? which functions to turn off? is there a special mode i didn't find? could an even better sd card like a sandisk extreme pro may help? anything else? of course thats not the first oproblem i have. osmand is the most feature rich but also most unstable navigation app i ever had, but i still love it and would like to have it back.

**Comment by DmitryAlexei** [comment]
Please see our docs about sd cards https://osmand.net/docs/user/troubleshooting/maps-data#maps-slowly-loading-on-android-11-12-sd-card

You can also try to disable everything you don't need for navigation in your driving profile (these preferences will affect only driving, other profiles will remain the same):
* you can disable 3d relief, contour lines, hillshades and slopes
* in menu > configure map > details you can disable unneeded details
* in menu > configure map > hide try to mark unneeded objects (boundaries, house numbers, underground objects, POI labels and POI icons, etc.)
* set map magnifier by 100% (long press + or - on the map screen) (this may be the main issue) 
* you can also disable autozoom (navigation > settings > navigation settings > map during navigation > auto zoom) 
* and use only vector maps. Online maps may consume additional power, especially if they are not previously downloaded.

**Comment by ulph0** [comment, marked as answer]
Thank you, @DmitryAlexei ! Unfortunately I didn't find all of that settings. I switched anything to the lowest setting and switched of all plugins but that didn't help. But maybe I found another solution: In Settings/OsmAnd-Settings I switched to Map-Rendering-Engine 1. that seemed to rise stability. I have to try it now but it looks good. It would be great osmand had the manual or automatic option to switch off/disable all memory/crash-critical functions/settings for older devices. Because as you know, the settings are a jungle.
